bigtop-tests/test-artifacts/hbase/src/main/groovy/org/apache/bigtop/itest/hbase/smoke/TestImportTsv.groovy

```diff
@@ -89,7 +92,8 @@ public class TestImportTsv {
     // delete data and junk from HDFS
     sh.exec("hadoop fs -rmr -skipTrash $DATADIR1",
       "hadoop fs -rmr -skipTrash $DATADIR2",
-      "hadoop fs -rmr -skipTrash /user/$USER/partitions_*");
+      "hadoop fs -rmr -skipTrash /user/$USER/hbase*",
```

Comment:
Got it and updated. I'm not sure about the context of using 'rmr', so I leave it unchanged in initial commit. :P